sys/crypto/chacha20poly1305.c

```diff
@@ -141,9 +127,8 @@ void chacha20poly1305_encrypt(uint8_t *cipher, const uint8_t *msg,
                               size_t msglen, const uint8_t *aad, size_t aadlen,
                               const uint8_t *key, const uint8_t *nonce)
 {
-    chacha20poly1305_ctx_t ctx;
-    _xcrypt(&ctx, key, nonce, msg, cipher, msglen);
-    crypto_secure_wipe(&ctx, sizeof(ctx));
```

Comment:
Do we still have the secure wipe of the ctx somewhere?

Reply:
yes, in the chacha20_finish function, which is called by the chacha20_encrypt_decrypt() function